### DISCUSSION TITLE
How to get the dependency of a atom

### DISCUSSION THREAD

**Discussion by himself65** [pmndrs/jotai#2009] [General] [answered]
```js
const a = atom(1)
const b = atom(2)

let c
//#region assume this atom is in a black box
c = atom(get => get(a) + get(b))
//#region

const v = store.get(c)
//How to get the deps of atom c?
// store.getDeps() like API
```

**Comment by dai-shi** [comment, marked as answer]
Such APIs are not exposed as it should be implementation details.
However, it's sometimes useful in development, and we have undocumented dev-only APIs.
https://github.com/pmndrs/jotai/blob/a9a639f4589c660b1f0723c536bf69d655bd71be/src/vanilla/store.ts#L741
https://github.com/pmndrs/jotai/blob/a9a639f4589c660b1f0723c536bf69d655bd71be/src/vanilla/store.ts#L758
